controllers/admin/AdminShopController.php

```diff
@@ -506,7 +507,8 @@ public function renderForm()
             'desc' => $this->trans('By selecting associated categories, you are choosing to share the categories between shops. Once associated between shops, any alteration of this category will impact every shop.', [], 'Admin.Shopparameters.Help'),
         ];
 
-        $themes = (new ThemeManagerBuilder($this->context, Db::getInstance()))
+        $themes = (SymfonyContainer::getInstance())
```

Comment:
I don't understand why you trust the SymfonyContainer here, but not in AdminStatsController :thinking:

Reply:
fixed